src/Synapse/Validator/Constraints/I18n/BelongsToEntity.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Synapse\Validator\Constraints\I18n;
+
+use Synapse\Validator\Constraints\BelongsToEntity as ParentConstraint;
+
+class BelongsToEntity extends ParentConstraint
+{
+    public $message = 'DOES_NOT_BELONG_TO_ENTITY';
```

Comment:
It would also be good to prefix all of these constants with `VALIDATION_`.  That would make it easy to find all validation messages in the codebase for translation